src/pages/iou/MoneyRequestAmountForm.tsx

```diff
@@ -144,7 +148,7 @@ function MoneyRequestAmountForm(
     }, [isFocused, wasFocused]);
 
     const initializeAmount = useCallback((newAmount: number) => {
-        const frontendAmount = newAmount ? CurrencyUtils.convertToFrontendAmount(newAmount).toString() : '';
+        const frontendAmount = newAmount ? CurrencyUtils.convertToFrontendAmountAsString(newAmount).toString() : '';
```

Comment:
```suggestion
        const frontendAmount = CurrencyUtils.convertToFrontendAmountAsString(newAmount);
```

Reply:
this suggestion will cause [this issue](https://github.com/Expensify/App/issues/39537) again

Reply:
Isn't this already handled in `convertToFrontendAmountAsString`?
```js
  if (amountAsInt === null || amountAsInt === undefined) {
        return '';
  }
```

Reply:
Not the same: 

`newAmount ?` the question mark here checks if the value is `null`, `undefined` and not `0` so in case the value is `0` it goes to the else condition which returns an empty string `""`